pkgs/development/libraries/ipu6-camera-hal/default.nix

```
@@ -52,16 +52,20 @@ stdenv.mkDerivation {
     gst_all_1.gst-plugins-base
   ];
 
-  preFixup = ''
-    ls -lah $out/lib/pkgconfig/
-    sed -Ei \
-      -e "s,^prefix=.*,prefix=$out," \
-      -e "s,^exec_prefix=.*,exec_prefix=''${prefix}," \
-      -e "s,^libdir=.*,libdir=''${prefix}/lib," \
-      -e "s,^includedir=.*,includedir=''${prefix}/include/libcamhal," \
-      $out/lib/pkgconfig/libcamhal.pc
+  postPatch = ''
+    substituteInPlace src/platformdata/PlatformData.h \
+      --replace '/usr/share/' "$out/share/"
```

```suggestion
      --replace '/usr/share/' "${placeholder "out"}/share/"
```

The `$out` variable does not exist before `installPhase`.

---

done